CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# Contributing to OpenVINO™ Intel® NPU Compiler
+
+We welcome contributions to the **OpenVINO™ Intel® NPU Compiler** project from both Intel employees and the open-source community.  
+This document provides guidelines to help you make successful contributions and ensure a smooth review and integration process.
+If you want to contribute to OpenVINO NPU plugin or have general questions about OpenVINO Toolkit contributions, refer to the [OpenVINO Contributing Guide](https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING.md)
+
+## Table of Contents
+1. [Ways to Contribute](#ways-to-contribute)  
+2. [Code Contribution Process](#code-contribution-process)  
+3. [Pull Request Requirements](#pull-request-requirements)  
+4. [Review Process](#review-process)  
+5. [Responsibilities](#responsibilities)  
+6. [Additional Resources](#additional-resources)
+
+
+## Ways to Contribute
+
+### Report Bugs or Propose Features
+
+- If you experience unexpected behavior or have ideas for improvement:
+  - Submit a [GitHub issue](https://github.com/openvinotoolkit/npu_compiler/issues) with a clear description, logs, and steps to reproduce.
+  - Or join the OpenVINO [GitHub Discussions](https://github.com/openvinotoolkit/openvino/discussions) if it's exploratory or architectural.
+
+### Suggest Architecture Improvements
+
+- For major proposals or POCs, open a **Draft Pull Request** and mark it as such to initiate discussion without triggering full review flow.
```

Comment:
> mark it as such

Does it mean to add a particular label to this draft PR?

Reply:
[DRAFT] or [POC] tag in a PR title